compiler/z/codegen/OMRMachine.hpp

```diff
@@ -640,7 +639,7 @@ class OMR_EXTENSIBLE Machine : public OMR::Machine
       }
    bool supportLockedRegisterAssignment();
 
-   TR::RealRegister *getRegisterFile(int32_t i);
+   TR::RealRegister *realRegister(int32_t rn) { return _registerFile[(TR::RealRegister::RegNum)rn]; }
```

Comment:
I don't think this is necessary. Won't we pick this API up from the extensible class? They are identical (modulo some casting needed for the argument).